### PR TITLE
Create Cucumber test environment end test.

### PR DESCRIPTION
**Pull request by Wis13**
[Trello](https://trello.com/c/o6RhtZF2/1009-f-000001-create-an-environment-for-cucumber)

### CODE REVIEW

File: src/test/java/runner/CucumberDriver.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package runner;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.Scenario;
+import org.openqa.selenium.WebDriver;
+
```
Comment (ke-de-ve) [inline src/test/java/runner/CucumberDriver.java:7]: delete row

File: src/test/java/runner/CucumberDriver.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+public class CucumberDriver {
+
+    private static WebDriver driver;
+
```
Comment (ke-de-ve) [inline src/test/java/runner/CucumberDriver.java:12]: delete row

File: src/test/java/runner/CucumberDriver.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    @Before
+    public static void before(Scenario scenario){
+        driver = BaseUtils.createDriver();
+
+        ProjectUtils.get(driver);
+        ProjectUtils.login(driver);
+        BaseTest.clearData();
+
+
```
Comment (ke-de-ve) [inline src/test/java/runner/CucumberDriver.java:22]: delete rows

File: src/test/java/runner/CucumberDriver.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    @After
```
Comment (ke-de-ve) [inline src/test/java/runner/CucumberDriver.java:24]: insert row

File: src/test/resources/cucumber/folder_test.feature [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created by Vadym Koroliuk at 12/14/2022
+Feature: Folder Test
+  # Enter feature description here
```
Comment (ke-de-ve) [inline src/test/resources/cucumber/folder_test.feature:3]: remove comment

File: src/test/resources/cucumber/folder_test.feature [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+  Scenario: Create new Folder
+    Given Go to home page
+    When Click on link New Item
+    And Enter an item name
+    And Select Folder from list and click Ok
+    And click Save
+    Then Result: folder was created
+    # Enter steps here
```
Comment (ke-de-ve) [inline src/test/resources/cucumber/folder_test.feature:12]: remove comment

**Review by ke-de-ve — APPROVED** [review]

**Review by Eugenezzz — APPROVED** [review]

**Pull request merged**